DataConnectors/Qualys VM/requirements.psd1

```diff
@@ -0,0 +1,7 @@
+# This file enables modules to be automatically managed by the Functions service.
```

Comment:
What's changed here? This is same as the profile.ps1 file that's part of PR#809 that we merged recently - file at
https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Qualys%20VM/requirements.psd1

Reply:
no changes here, these are just generic/default files created with every function app. Also a requirement for the error we recieved